### PR TITLE
component/menu: research and testing libraries, ending up using react…

### PR DESCRIPTION
**Pull request by khamui**
… elements

After reading and trying some libraries for `backdrops` and `overlays`, i found `react native elements` is what provides a similar behavior how we want it.

Nevertheless it will be not easy to get an accurate result compared to the designs. It will be a simpler version of the menu.

### CODE REVIEW

File: containers/Ideas/Idea.js [hunk 1]
```diff
@@ -15,67 +16,97 @@ export const Idea = (props) => {
...
-        <Text style={styles.text}>
-          Reference No.: {params.reference_number || 'n/a'}
-        </Text>
-      </View>
-      <View style={styles.bottomActionsContainer}>
-        <View style={styles.ratingButtons}>
-          <ButtonCounter
-            icon={<Icon name='arrow-up' size={18} />}
-            counter={params.positive_rating_count}
+    <>
```
Comment (khamui) [inline containers/Ideas/Idea.js:47]: except of last line  `<Menu...` it is just indentation

File: components/Menu.js [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
 };
+
+const styles = StyleSheet.create({
+  backdrop: {
+    backgroundColor: 'rgba(77, 77, 77, 0.9)'
+  },
+  list: {
+    alignItems: 'center'
+  }
+});
```
Comment (phillimorland) [inline components/Menu.js:27]: We are currently using seperate style sheets for components, I think we should stick with doing it one way or the other, which way do you prefer?

File: components/MenuItem.js [hunk 3]
```diff
@@ -1,19 +1,67 @@
...
+  cancelContent: {
+    alignItems: 'center',
+  },
+  cancelButton: {
+    backgroundColor: 'transparent'
+  },
+  cancelButtonText: {
+    color: 'white'
+  }
+});
```
Comment (phillimorland) [inline components/MenuItem.js:67]: same as above

File: containers/Ideas/Idea.js (hunk 1 above)
Comment (phillimorland) [reply containers/Ideas/Idea.js:47]: Doe you mean how different file is?
Also do we need the console.log?

**Review by phillimorland — COMMENTED** [review]
just a couple of questions but other then that, looking good :)

**Pull request merged**